fontbakery: 0.12.7 -> 0.12.9

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for fontbakery is: Tool for checking the quality of font projects

meta.homepage for fontbakery is: https://github.com/googlefonts/fontbakery




###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes





###### Impact

<b>Checks done</b>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 0.12.9 with grep in /nix/store/2mfgbc5p7f52xp5zalrxv6dz85na6li0-python3.12-fontbakery-0.12.9
- found 0.12.9 in filename of file in /nix/store/2mfgbc5p7f52xp5zalrxv6dz85na6li0-python3.12-fontbakery-0.12.9

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
4 total rebuild path(s)

4 package rebuild(s)

First fifty rebuilds by attrpath

fontbakery
python311Packages.fontbakery
python312Packages.fontbakery
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/2mfgbc5p7f52xp5zalrxv6dz85na6li0-python3.12-fontbakery-0.12.9 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A fontbakery https://github.com/r-ryantm/nixpkgs/archive/a6e454f2c097ad761c23120fa709091abafeac7a.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/a6e454f2c097ad761c23120fa709091abafeac7a#fontbakery
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/2mfgbc5p7f52xp5zalrxv6dz85na6li0-python3.12-fontbakery-0.12.9
ls -la /nix/store/2mfgbc5p7f52xp5zalrxv6dz85na6li0-python3.12-fontbakery-0.12.9/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>fontbakery (python312Packages.fontbakery)</li>
    <li>fontbakery.dist (python312Packages.fontbakery.dist)</li>
    <li>python311Packages.fontbakery</li>
    <li>python311Packages.fontbakery.dist</li>
  </ul>
</details>

---

###### Maintainer pings

cc @danc86 for [testing](https://github.com/ryantm/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

r-ryantm committed:
fontbakery: 0.12.7 -> 0.12.9

fabaff commented:
Result of `nixpkgs-review pr 332360` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>fontbakery (python312Packages.fontbakery)</li>
    <li>fontbakery.dist (python312Packages.fontbakery.dist)</li>
    <li>python311Packages.fontbakery</li>
    <li>python311Packages.fontbakery.dist</li>
  </ul>
</details>

fabaff committed:
python312Packages.fontbakery: add changelog to meta

fabaff committed:
python312Packages.fontbakery: refactor

- disable on unsupported Python releases
- update ordering for easier reviewing

fabaff commented:
Result of `nixpkgs-review pr 332360` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>fontbakery (python312Packages.fontbakery)</li>
    <li>fontbakery.dist (python312Packages.fontbakery.dist)</li>
    <li>python311Packages.fontbakery</li>
    <li>python311Packages.fontbakery.dist</li>
  </ul>
</details>

fabaff reviewed (approved):
Change looks good to me.
